faucet/valve.py

```diff
@@ -932,6 +932,17 @@ def lacp_handler(self, now, pkt_meta):
                     ofmsgs.append(valve_of.packetout(pkt_meta.port.number, pkt.data))
                 pkt_meta.port.dyn_last_lacp_pkt = lacp_pkt
                 pkt_meta.port.dyn_lacp_updated_time = now
+                other_lag_ports = [
+                    port for port in self.dp.ports.values()
+                    if port.lacp == pkt_meta.port.lacp and port.dyn_last_lacp_pkt]
+                actor_system = pkt_meta.port.dyn_last_lacp_pkt.actor_system
+                for other_lag_port in other_lag_ports:
+                    other_actor_system = other_lag_port.dyn_last_lacp_pkt.actor_system
+                    if actor_system != other_actor_system:
+                        self.logger.error(
+                            'actor system mismatch %s: %s, %s %s' % (
```

Comment:
Should this line mention LACP protocol by name for those unfamiliar with what a LACP actor is?

Reply:
Added.